src/null-safety/index.md

```diff
@@ -115,6 +115,24 @@ To resolve these issues, check for
 of any packages you installed from pub.dev, and [migrate](#migrate) all
 of your own source code to use sound null safety.
 
+As of late January 2023, **Dart 3 alpha** is available from the Dart dev channel
+and the Flutter master channel; see [the download page][] for details.
+We recommend you test your code for Dart 3 compatibility using that release:
+
+```terminal
+$ dart --version                     # make sure this reports 3.0.0-151.0.dev or higher
+$ dart pub get                       # this should resolve without issues
```

Comment:
If you're going to specify the `flutter` equivalent command for `analyze`, should probably do so here to:

```suggestion
$ dart pub get / flutter pub get     # this should resolve without issues
```